src/app/cards/edit/[id]/Form.tsx

```diff
@@ -0,0 +1,242 @@
+        >
+            <TopNav left={getLeftBtn()} right={getRightBtn()} />
+            <div className="w-10/12 justify-self-center flex flex-col items-center gap-1.5">
+                <p className="text-xs text-danger-600 h-4">{errorMsg}</p>
+                <div className="w-full flex flex-col items-center">
+                    <p className="text-xs">FRONT</p>
+                    <input
+                        id="image_front"
+                        name="image_front"
+                        type="file"
+                        accept="image/*"
+                        onChange={(event) => handleImgChange(event)}
+                        disabled={loading}
+                        className="h-0"
+                    />
+                    <label
+                        htmlFor="image_front"
+                        className={`w-36 h-24 overflow-hidden rounded-xl border ${
+                            !loading && 'cursor-pointer'
+                        }`}
+                    >
+                        <img
+                            src={card.image_front || 'https://placehold.co/600x400?text=Required'}
+                            alt="Image Preview for Front side"
+                            className="w-full h-full object-cover object-center"
+                        />
+                    </label>
+                </div>
+                <div className="w-full flex flex-col items-center pb-1">
+                    <p className="text-xs">BACK</p>
+                    <input
+                        id="image_back"
+                        name="image_back"
+                        type="file"
+                        accept="image/*"
+                        onChange={(event) => handleImgChange(event)}
+                        disabled={loading}
+                        className="h-0"
+                    />
+                    <label
+                        htmlFor="image_back"
+                        className={`w-36 h-24 overflow-hidden rounded-xl border ${
+                            !loading && 'cursor-pointer'
+                        }`}
+                    >
+                        <img
+                            src={card.image_back || 'https://placehold.co/600x400?text=NoImage'}
+                            alt="Image Preview for Back side"
+                            className="w-full h-full object-cover object-center"
+                        />
+                    </label>
+                </div>
+
+                <div className="w-full flex flex-col gap-0.5">
+                    <label htmlFor="name" className="text-xs self-start">
+                        CARD NAME
+                    </label>
+                    <input
+                        id="name"
+                        type="text"
+                        value={card.name}
+                        onChange={(event) => setCard({ ...card, name: event.target.value })}
```

Comment:
[NIT] The `(event) => setCard({ ...card, name: event.target.value })` repeated many times in this file. Shall we move it out as a function?

Reply:
Fixed